ssc/Pos/Ssc/GodTossing/Instance.hs

```diff
@@ -0,0 +1,26 @@
+{-# LANGUAGE TypeFamilies #-}
+
+module Pos.Ssc.GodTossing.Instance
+       ( -- * Instances
+         -- ** instance SscHelpersClass
+       ) where
+
+import           Universum
+
+import           Pos.Binary.Core                    ()
+import           Pos.Binary.GodTossing              ()
+import           Pos.Core                           (HasConfiguration)
+import           Pos.Ssc.Class.Helpers              (SscHelpersClass (..))
+import           Pos.Ssc.Core                       (defaultSscPayload,
+                                                     stripSscPayload)
+import           Pos.Ssc.GodTossing.Functions       (sanityChecksSscPayload)
+import           Pos.Ssc.VerifyError                (SscVerifyError (..))
+
+instance HasConfiguration => SscHelpersClass where
```

Comment:
Remove `SscHelpersClass` in the current iteration, please. It has no parameters and its instance defined in the same package, so you're not going to hit any circular dependencies, should be rather painless.